src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -31,11 +32,13 @@ public static class Options
             public const string SingleTopLevelClient = "single-top-level-client";
             public const string AttachDebuggerFormat = "{0}.attach";
             public const string ProjectFolder = "project-folder";
+            public const string ExistingProjectfolder = "existing-project-folder";
             public const string ProtocolMethodList = "protocol-method-list";
             public const string SkipSerializationFormatXml = "skip-serialization-format-xml";
             public const string DisablePaginationTopRenaming = "disable-pagination-top-renaming";
             public const string SuppressAbstractBaseClasses = "suppress-abstract-base-class";
             public const string UnreferencedTypesHandling = "unreferenced-types-handling";
+            public const string AmbiguityHandling = "ambiguity-handling";
```

Comment:
Is `ambiguity-handling` not a command line option? If so, how autorest configure this?

Reply:
I don't think autorest has such requirement. Others don't have generated convenience method. At the very least, autorest configures it in the `autorest.md`.